Gemfile

```diff
@@ -29,12 +29,16 @@ gem 'jbuilder', '~> 2.5'
 # gem 'redis', '~> 3.0'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
-
+gem "jquery-rails"
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 gem 'devise', '4.3.0'
 gem "paperclip", "~> 5.2.1"
 gem "ckeditor"
+gem "cocoon"
+gem "chosen-rails"
+gem 'jquery-turbolinks'
```

Comment:
Why are we using this one?
it seems to be deprecated.

Reply:
Yes Sir, Its not needed now I forgot to remove I added this for the turbolink issue which I was facing yesterday.